.github/release.yml

```diff
@@ -0,0 +1,36 @@
+# .github/release.yml
+
+changelog:
+  exclude:
+    labels:
+      - release-note/ignore-for-release
+    authors:
+      - octocat
+  categories:
+    - title: Exciting New Features 🎉
+      labels:
+        - release-note/new-feature
+    - title: Enhancement 🚀
+      labels:
+        - release-note/enhancement
+    - title: Component updates ⬆️
+      labels:
+        - release-note/update
+    - title: Docs update 🗄️
```

Comment:
@OrlinVasilev I'm not sure if we need the docs update feature here. Or, rather, i dont think this will end up pulling in anything from this repo, bc all docs change PRs are made in the website repo. 

Is there a way to make this automation across repos? 

Does it make sense to plan on adding in docs updates to release notes manually for now, until we can figure out the automation?

Reply:
that's just a label definition which will help us in the future, cross repo is something that I started looking today will continue on Monday, but in general we should have as many as possible ways not to miss any change or update!